DOM_creation/Lesson4/main.js

```diff
@@ -0,0 +1,18 @@
+const ul = document.getElementById("js-ul");
+const fragment = new DocumentFragment();
+const items = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
```

Comment:
IMO
It is better to use concrete names for variables. 
It's easier to understand what kind of value is stored in the variable.

I recommend this article.
https://github.com/mitsuruog/clean-code-javascript#variables

Reply:
Hi Mai-san,
Thank you for your feedback and sharing the article.
I read it and changed the variable name, yet I am still not quite sure the updated variable name is more precise or not.
What name would you use to describe a group of attributes like the one in this lesson? 

Kindly check my latest commit

Reply:
Hi Saki-san,
Thank you for the correction.

Sorry, I may not have communicated it well.
It is more reader-friendly to use specific names for variables.
A person reading this code for the first time will want to know what kind of data items is.

What kind of data is items?

For example, the name `features`.
This is because it contains data about features such as messages and bookmarks that you often see on social media.

This is more specific than ITEMS, but not the best name.
If you can think of a better name, let us know!

Reply:
Thank you for your advice Mai-san.
I changed it to features as your suggestions. 
I searched some examples and majority of articles says that the rules of thumb is to use "plural" form for a variable name to describe and array. Also using "list" could be another option, for example "featureList". 

I find this article is well explained.
[10 JavaScript Naming Conventions Every Developer Should Know](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)